cmake/sdlcpu.cmake

```diff
@@ -3,6 +3,10 @@ function(SDL_DetectTargetCPUArchitectures DETECTED_ARCHS)
   set(known_archs EMSCRIPTEN ARM32 ARM64 ARM64EC LOONGARCH64 POWERPC32 POWERPC64 X86 X64)
 
   if(APPLE AND CMAKE_OSX_ARCHITECTURES)
+    list(LENGTH CMAKE_OSX_ARCHITECTURES _num_arches)
+    if(_num_arches GREATER 1)
+      set(APPLE_MULTIARCH 1 PARENT_SCOPE)
+    endif()
```

Comment:
This pr will work, but I'd prefer to not do this in `sdlcpu.cmake`. This file is also used in the Android and MSVC release assets, and should not define anything else then its interface.

Reply:
OK, I moved it to top-level cmake script.